keras_cv/src/models/stable_diffusion_v3/model_sampling_discrete_flow_test.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2024 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+from keras import ops
```

Comment:
will need to update the import from 
`from keras import ops` -> `from keras_cv.src.backend import ops`

Reply:
Fixed